chats.py

```diff
@@ -17,3 +17,17 @@ def pieraksti_zinju(dati):
     json_data["laiks"] = datetime.now()
     with open(LOGFAILS, "a", newline="", encoding="utf-8") as f:
         f.write(json.dumps(json_data, ensure_ascii=False) + "\n")
+
+GARASTAVOKLIS = "mood.txt"
```

Comment:
Piedāvāju garastāvokļa saistīto kodu iznest jaunā failā: garastavoklis.py, kur attiecīgi veikt visu loģiju, un importēt failu iekš main.py, kā tas ir chats.py  gadījumā.